addons/mail/static/src/core/common/chat_bubble.scss

```diff
@@ -0,0 +1,76 @@
+
+.o-mail-ChatBubble {
+    height: auto !important;
```

Comment:
overall, now the js and design team is gradually making everything bootstrap class defined. So if we have the chance to do so, we should follow the trend as well? So that's easier to maintain the code.

Reply:
I understand to prefer using bootstrap classes to avoid writing any SCSS, but sometimes it's stupid to write the same classnames in all occurrences of multiple targeted elements.

There are 3 UI elements that have `o-mail-ChatBubble`. One is an actual chat bubble, the 2 others want to look the same, hence I've just reused the classname. It's worse maintenance to ensure they have the same multiple bootstrap classes when you can just share it automatically by defining in SCSS once.